src/utilities/__tests__/findDeprecatedUsages-test.js

```diff
@@ -1,5 +1,7 @@
 // @flow strict
 
+/* eslint-disable import/no-deprecated */
+
```

Comment:
If you implement an old function on top of a new validation rule you can simply remove all tests and disable coverage on `findDeprecatedUsages.js`